backend/internal/db/game_test.go

```diff
@@ -0,0 +1,117 @@
+package db
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/jak103/powerplay/internal/models"
+)
+
+func (s *dbTestingSuite) CreateGameHelper() models.Game {
+	// Create initial data
+	season := models.Season{Name: "Season 1"}
+	dbSeason, err := s.session.CreateSeason(&season)
+	s.Nil(err)
+	fmt.Println("Created Season ID:", dbSeason.ID)
```

Comment:
Rather than printing the results, try `s.Equal(uint(1), dbSeason.ID)`. This will compare the results and report if they are different.

Reply:
This goes for all `Println` statements. The tests we create should not be visibily verified, but verified in the code with the results reported on build. Dr. Christensen set this up in his repository and on the Cloud environment.